assets/css/index.css

```diff
@@ -121,6 +121,10 @@ section {
   margin-bottom: 1em;
   font-size: 1.5rem;
 }
+.button-container .sbtn :hover{
```

Comment:
Please remove this as well. I just need the changes in `index.html`

Reply:
yes , I am removing it